validation/src/arbor-rc-exp2syn-spike/arbor-rc-exp2syn-spike.cpp

```diff
@@ -139,60 +140,22 @@ domain_decomposition trivial_dd(const recipe& r) {
     };
 }
 
-struct nc_error: std::runtime_error {
-    nc_error(const char* fn, int st):
-        std::runtime_error(std::string(fn)+": "+std::string(nc_strerror(st))),
-        call(fn),
-        status(st) {}
-
-    int status;
-    std::string call;
-};
-
-struct arg_data {
-    std::string output;
-    paramset params = default_parameters;
-};
-
-void common_parse_arguments(char** argv, arg_data& args);
-void write_netcdf_traces(const char* path, const arb::trace_data<double>& v0, const std::vector<double>& spikes, const std::vector<double>& delay, const paramset& scalars);
-
-int usage(char* argv0) {
-    char* basename = std::strrchr(argv0, '/');
-    basename = basename? basename+1: argv0;
-
-    std::cerr << "Usage: " << basename << " OUTFILE [ PARAM=VALUE ... ]\n";
-    return 1;
-}
-
 int main(int argc, char** argv) {
-    paramset params{default_parameters};
-    const char* output = argv[1];
-
-    if (!output) return usage(argv[0]);
-    try {
-        for (auto a = argv+2; *a; ++a) {
-            if (char* eq = std::strrchr(*a, '=')) {
-                params[std::string{*a, eq}] = std::stod(eq+1);
-            }
-            else return usage(argv[0]);
-        }
-    }
-    catch (...) {
-        return usage(argv[0]);
-    }
+    common_args A;
```

Comment:
How about `args` instead of `A`?

Reply:
Went with `A` not just because it was brief, but also because `args` it was visually confusing with `argc` and `argv`, and `params` was already used for the test parameters.